package.json

```diff
@@ -117,5 +117,8 @@
         "prerelease": true
       }
     ]
+  },
+  "dependencies": {
+    "@typescript-eslint/utils": "^5.56.0"
```

Comment:
This must be marked a dependency so that it's downloaded and included at runtime for the developer.

It's not _actually_ a dependency of HouseForm's output bundle.